core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -194,6 +194,24 @@ span.additional-metric-title {
   color: #777;
 }
 
+#additional-metrics {
+  float: left;
+  padding-bottom: 10px;
+}
+
+#links {
+  float: left;
+  padding-left: 100px;
```

Comment:
That's right, I'm just referring to "100px right" when I say absolute. It's not positioned absolutely vertically. And "inline" flow meaning, what you'd get just concatenating divs on a page rather than making one float right. That's the wrong term in HTML isn't it? I think "block" is what I'm looking for. I'm neutral on adding this in any event, so will let others pick it up or not.